pkg/restore/restore_test.go

```diff
@@ -2113,6 +2112,130 @@ func TestRestorePersistentVolumes(t *testing.T) {
 	}
 }
 
+type fakeResticRestorerFactory struct {
+	restorer         *resticmocks.Restorer
+	podVolumeBackups []*velerov1api.PodVolumeBackup
+}
+
+func (f *fakeResticRestorerFactory) NewRestorer(ctx.Context, *velerov1api.Restore) (restic.Restorer, error) {
+	return f.restorer, nil
+}
+
+type fakeResticRestorer struct {
```

Comment:
I don't think the `fakeResticRestorer` struct is used - is it? let's delete if not.